### ISSUE TITLE
Update content and documentation after transfer repo to WordPress organisation

### ISSUE THREAD

**Issue by rianrietveld** [WordPress/accessibility-team-handbook#12]
The content of the Accessibility Team Handbook is published on [make.wordpress.org/accessibility](https://make.wordpress.org/accessibility/handbook/). This handbook will be maintained in a GitHub repository instead of using the WordPress Admin.

The team prepared the content of this handbook in a repository of the 
[WordPress Accessibility](https://github.com/wpaccessibility) and transferred it to the WordPress organisation.

What needs updating:

Take https://github.com/WordPress/hosting-handbook as an example/template.

- README.md: like hosting-handbook
- Contributing.md: check if the info is still valid and update if needed. Run `wp accessibility-handbook gen-all` should be `wp accessibility-team-handbook gen-all`.
- Licence.md: is it GNU General Public License v3.0 or v2.0?
